custom-plugin-rules/require-bash-sample/README.md

```diff
@@ -0,0 +1,91 @@
+# Require Bash Sample
```

Comment:
https://github.com/Redocly/redocly-cli-cookbook/pull/42/files#r1809495936
I left this comment on another similar PR. This doesn't feel so practical. Most people don't have `bash` samples and use `curl` for example (so "bash" is probably a rare/niche example for Redoc users). 

I think it might be nice since this PR contains two separate plugins to separate it into two PRs, because the default enum one would be easy to approve.

This one should should probably be modified. I think the plugin would be more useful if it accepted an array of languages in the configuration and gave an error or warning (depending on severity) based on the list matching.

This is also similar to #42 , and I think if this was made more flexible should replace it. That one has a lot of quirks and hardcodes errors and warnings instead of taking the severity from the configuration.

Reply:
@adamaltman For now, I removed the bash example rule entirely. When I get a chance, I'll see if I can whip something together that's better suited, but I don't want to block the default/enum rule from getting merged.